server/sttp-stub-server/src/main/scala/sttp/tapir/server/stub/SttpStubServer.scala

```diff
@@ -0,0 +1,125 @@
+package sttp.tapir.server.stub
+
+import sttp.client.monad.MonadError
+import sttp.client.monad.syntax._
+import sttp.client.testing.SttpBackendStub
+import sttp.client.{NothingT, Request, Response, SttpBackend}
+import sttp.model.StatusCode
+import sttp.tapir.internal.SeqToParams
+import sttp.tapir.server.internal.{DecodeInputs, DecodeInputsResult, InputValues}
+import sttp.tapir.server.{DecodeFailureContext, DecodeFailureHandling, ServerDefaults, ServerEndpoint}
+import sttp.tapir.{DecodeFailure, Endpoint, EndpointInput}
+
+trait SttpStubServer {
+
+  implicit class RichStubServerEndpoints[F[_]](
+      endpoints: Iterable[ServerEndpoint[_, _, _, _, F]]
+  ) {
+    def toBackendStub(implicit me: MonadError[F]): SttpBackend[F, Nothing, NothingT] = {
```

Comment:
so there are two ways to use the stub now,yes? one using `.toBackendStub` on server endpoints, and the other using `when`/`then`?